lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -138,11 +137,9 @@ def generate_log_for_all_tags
 
       log = generate_unreleased_section
 
```

Comment:
This is the main change. Using the hash with left-tag, right-tag to create sections